src/main/java/kitchenpos/menu/domain/MenuProducts.java

```diff
@@ -0,0 +1,25 @@
+package kitchenpos.menu.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+
+@Embeddable
+public class MenuProducts {
+    @OneToMany(mappedBy = "menu", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    private List<MenuProduct> menuProducts = new ArrayList<>();
+
+    public MenuProducts(List<MenuProduct> menuProducts) {
+        this.menuProducts = menuProducts;
+    }
```

Comment:
이 부분도 방어적 복사해도 좋을 것 같아요 ㅎㅎ

```suggestion
    private List<MenuProduct> menuProducts;

    public MenuProducts(List<MenuProduct> menuProducts) {
        this.menuProducts = new ArrayList<>(menuProducts);
    }
```